widget/ui/src/constants/index.ts

```diff
@@ -0,0 +1,6 @@
+export const UI_ID = {
+  RANGO_SWAP_FROM_INPUT_CONTAINER_ID: 'rango-swap-from-input-container',
```

Comment:
You can remove `RANGO_` prefix from these values.

Reply:
Done